src/OnChain.ts

```diff
@@ -209,7 +209,7 @@ export const OnChainMixin = (superclass) =>
           onchainLogger = onchainLogger.child({ onUs: false })
 
           if (!this.user.oldEnoughForWithdrawal) {
-            const error = `New accounts have to wait ${this.config.limits.oldEnoughForWithdrawalLimit()}h before withdrawing`
+            const error = `New accounts have to wait ${this.config.limits.oldEnoughForWithdrawalLimitHours}h before withdrawing`
```

Comment:
Following on from the previous comment, this could become `oldEnoughForWithdrawalHours`

Reply:
**_Done: https://github.com/GaloyMoney/galoy/pull/393/commits/b01e2283937c65b6fe45bf22442a97c6ba3c466d_**